runtime/src/snapshot_utils.rs

```diff
@@ -958,6 +974,96 @@ where
     Ok((bank, timings))
 }
 
+/// Rebuild bank from snapshot archives.  This function searches `snapshot_archives_dir` for the
+/// highest full snapshot and highest corresponding incremental snapshot, then rebuilds the bank.
+#[allow(clippy::too_many_arguments)]
+pub fn bank_from_latest_snapshot_archives(
```

Comment:
nit: want to communicate this function also verifies the hash

What do you think of `get_and_verify_bank_from_latest_snapshot_archives()`, too verbose?